src/trackers/helpers/sharedData.js

```diff
@@ -33,6 +34,14 @@ function _readEntities () {
     }, {})
 }
 
+// option list of top example sites to include in tracker files
+function _getTopExampleSites (config) {
+    if (!config.topExampleSites) {
+        return null
+    }
+    return new Set(JSON.parse(fs.readFileSync(config.topExampleSites, 'utf8')))
```

Comment:
can we document how this file should look like (structure)?

Reply:
Added a comment to the config file